src/main/java/org/broadinstitute/hellbender/tools/exome/segmentation/PerformAlleleFractionSegmentation.java

```diff
@@ -18,7 +19,7 @@
  * Groups contiguous targets with the same minor allele fraction for a single sample.
  *
  *  <p>
- *     The --tumorHets file is from {@link GetBayesianHetCoverage}, preferably, but can also come from {@link org.broadinstitute.hellbender.tools.exome.GetHetCoverage}.
+ *     The --tumorHets file is from {@link org.broadinstitute.hellbender.tools.exome.GetBayesianHetCoverage}, preferably, but can also come from {@link org.broadinstitute.hellbender.tools.exome.GetHetCoverage}.
```

Comment:
I think you might have to change the dropdown from "If not already imported" to "Never, use short name and add import" if you want the IntelliJ autocorrect to use the short name.